### DISCUSSION TITLE
[UNSTABLE] Respawns in deathmatch (and KOTH?) happen on the docked submarine.

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14284] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When respawning in deathmatch and KOTH, players will spawn on the docked submarine instead of the station.

### Reproduction steps

1. Start a game of deathmatch or KOTH
2. Die
3. Respawn
4. Welcome to the submarine.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and fix tested to be working correctly. Fix coming in the next Unstable build. Closing.
